src/wp-includes/block-supports/typography.php

```diff
@@ -113,10 +113,10 @@ function wp_apply_typography_support( $block_type, $block_attributes ) {
 
 	$typography_block_styles = array();
 	if ( $has_font_size_support && ! $should_skip_font_size ) {
-		$preset_font_size                    = array_key_exists( 'fontSize', $block_attributes )
+		$preset_font_size                        = array_key_exists( 'fontSize', $block_attributes )
 			? "var:preset|font-size|{$block_attributes['fontSize']}"
 			: null;
-		$custom_font_size                    = isset( $block_attributes['style']['typography']['fontSize'] )
+		$custom_font_size                        = isset( $block_attributes['style']['typography']['fontSize'] )
```

Comment:
Tiny nit: it looks like there's something odd happening with the whitespace here. Is the line below `$typography_block_styles['fontSize']` indented one tab too far? (I.e. instead of adding extra whitespace on this line and the one above, can we remove the tab from `$typography_block_styles['fontSize']` instead?

Reply:
Good catch. Fixed.